lib/liquid/block_body.rb

```diff
@@ -93,10 +93,10 @@ def render(context)
         rescue MemoryError => e
           raise e
         rescue UndefinedVariable, UndefinedDropMethod, UndefinedFilter => e
-          context.handle_error(e, token.line_number)
+          context.handle_error(e, token.line_number, token.raw)
           output << nil
         rescue ::StandardError => e
-          output << context.handle_error(e, token.line_number)
+          output << context.handle_error(UnhandledError.new(e), token.line_number, token.raw)
```

Comment:
:-1: to what he was suggesting there, which is basically ignoring the bug in liquid by treating it as a bug in the template